src/providers/ad/ad_gpo.c

```diff
@@ -100,7 +100,6 @@
 /* If INI_PARSE_IGNORE_NON_KVP is not defined, use 0 (no effect) */
 #ifndef INI_PARSE_IGNORE_NON_KVP
 #define INI_PARSE_IGNORE_NON_KVP 0
-#warning INI_PARSE_IGNORE_NON_KVP not defined.
```

Comment:
I am not sure whether it is the latest version of patch set from #50 because this warning is intentionally here.
And was removed from #50

https://github.com/SSSD/sssd/pull/50#discussion_r87219308